### PR TITLE
[MMC-2244] Move SDK to v1.0

### PR DESCRIPTION
**Pull request by katie-ham**
### Context / Description
Update version to 1.0

### Steps to Test / Edge Cases
Confirm version is now 1.0

### Known Issues
none

### Impact / Risk
low

### Reminders

- [x] [Checked for accessibility impact](https://github.com/mailchimp/MCM-iOS/wiki/Accessibility)

---

**JIRA: [MMC-2244](https://jira.mailchimp.com/browse/MMC-2244)**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0] - 2020-07-20
+
```
Comment (michael-patzer) [inline Changelog.md:8]: Let's add some kind of note here, even if it's simply to say that this is an official release with no notable changes from 0.1.6
